src/ResourceManager/Insights/Commands.Insights/Diagnostics/GetDiagnosticSettingCommand.cs

```diff
@@ -0,0 +1,50 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Threading;
+using Microsoft.Azure.Commands.Insights.OutputClasses;
+using Microsoft.Azure.Management.Insights.Models;
+
+namespace Microsoft.Azure.Commands.Insights.Diagnostics
+{
+    /// <summary>
+    /// Gets the logs and metrics for the resource.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "DiagnosticSetting"), OutputType(typeof(PSServiceDiagnosticSettings))]
+    public class GetDiagnosticSettingCommand : ManagementCmdletBase
```

Comment:
The name of these cmdlets seems odd. Shouldn't it be something like `Get-AzureRmDiagnosticSetting` or `Get-AzureRmInsightsDiagnosticSetting`?